### DISCUSSION TITLE
Will Gamera-4 be packaged soon?

### DISCUSSION THREAD

**Discussion by rmast** [hsnr-gamera/gamera-4#40] [Q&A] [answered]
Ubuntu 22.04 is now able to compile Gamera-4 and run didjvu on Python3.10 with the python3 branches of FriedrichFroebel a patch from me.

The biggest issue we'll have with getting it packaged again is the availability of the Gamera(-4) package.

For didjvu we've made all unittests run, however there has not been much testing as there are hardly any users/testers known.

I guess getting into the packaging-process of both and get it back in the debian-distributions would reveal whether there are still some blindspots left. Ubuntu 22.04 is scheduled for the 21st of April. When will Gamera-4 be packaged?

**Comment by cdalitz** [comment]
Thanks for the info about the upcoming Ubuntu release date. I will create a file release with a proper version number during this week so that you can include it.

There are currently two issues that I am aware of, but non of this is critical: Gamera still uses the old buffer protocol, and the genetic algorithm for optimizing features does not load, but his already was a problem with the last release of Gamera version 3. And as we have greedy feature selection implemented, there is no actual need to apply genetic optimization, because in all cases I tried it, the improvements over greedy feature selection are barely noticable.

**Comment by cdalitz** [comment, marked as answer]
Gamera 4.0.0 is now released. This is the first version that officially supports Python 3:
[https://github.com/hsnr-gamera/gamera-4/releases/tag/v4.0.0](https://github.com/hsnr-gamera/gamera-4/releases/tag/v4.0.0)
